src/item.cpp

```diff
@@ -2758,6 +2758,11 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
 
             const bionic_id bid = type->bionic->id;
 
+            if( bid->capacity > 0_mJ ) {
+                info.push_back( iteminfo( "CBM", _( "<bold>Power Capacity:</bold> " ), " <num> mJ",
```

Comment:
```suggestion
                info.push_back( iteminfo( "CBM", _( "<bold>Power Capacity:</bold>" ), " <num> mJ",
```
Don't need two spaces here.